icon/iiss/icstate/prepstatus_test.go

```diff
@@ -116,7 +116,7 @@ func TestPRepStatus_GetBondedDelegation(t *testing.T) {
 	bonded = big.NewInt(int64(999))
 	s.GetPRepStatus(addr1, true).SetBonded(bonded)
 	res = status1.GetBondedDelegation(0)
-	assert.Equal(t, 0, res.Cmp(big.NewInt(int64(0))))
+	assert.Equal(t, 0, res.Cmp(big.NewInt(int64(100998))))
```

Comment:
```suggestion
	assert.Equal(t, 0, res.Cmp(status1.GetVoted()))
```

Reply:
assert.Equal(t, 0, res.Cmp(big.NewInt(int64(100998))))
```suggestion
	assert.Equal(t, 0, res.Cmp(big.NewInt(status1.GetVoted().Int64())))
```

I reflected like above, thank you @goldworm